main.go

```
@@ -58,6 +58,8 @@ var (
 	autoGenerateCreatedDate       = os.Getenv("AUTO_GENERATE_CREATED_DATE")
 	limitOcrPages                 int // Will be read from OCR_LIMIT_PAGES
 	tokenLimit                    = 0 // Will be read from TOKEN_LIMIT
+	ocrEnableHOCR                 = os.Getenv("OCR_ENABLE_HOCR") == "true"
```

golangci-lint (1.64.8)

var `ocrEnableHOCR` is unused

(unused)